### PR TITLE
fix(docs-infra): fix get started button rendering

### PR DESCRIPTION
**Pull request by TzviPM**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #31454 

angularIoDisplayBug

## What is the new behavior?
angularIoDisplayBugSuggestedFixNoHeight

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/src/styles/1-layouts/_marketing-layout.scss [hunk 1]
```diff
@@ -246,11 +246,6 @@ section#intro {
 }
 
 .button.hero-cta {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  width: 184px;
-  height: 40px;
   padding: 0 24px;
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_marketing-layout.scss]: Can you also add some more padding on the sides? Maybe `padding: 0 34px;` (or even `padding 2px 34px 0;`)?

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx for jumping on to this, @tzvipm :100:
I had one suggestion, but otherwise lgtm 👍 

I, also, see you put effort into getting the commit message right 😍 (and did great :100:), but there's just a couple of details that need tweaking to completely adhere to [our guidelines][1]:

> fix(docs-infra): fix get started button rendering
>
> Fixes #31454

[1]: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

File: aio/src/styles/1-layouts/_marketing-layout.scss (hunk 1 above)
Comment (tzvipm) [reply aio/src/styles/1-layouts/_marketing-layout.scss]: Turns out the even vertical padding looks a bit off, so I opted for the latter suggestion :+1:

**Comment by tzvipm** [comment]
@gkalpak PR is updated with your suggested changes. Thanks for your help getting this moved along.

**Review by gkalpak — APPROVED** [review]
Looks great! Thx, @tzvipm :+1:

**Pull request closed without merging**